### PR TITLE
add rosconverter detectionrobot

### PR DESCRIPTION
**Pull request by adithya-gv**
## Description
Completed DetectionRobotMsg rosconverter for SSLBall

## Associated Issue
Issue #1588 

## Design Documents
N/A

### Test Case 1
1.Run soccer

Expected result: No regression expected

### CODE REVIEW

File: rj_utils/include/rj_utils/conversions.hpp [hunk 1]
```diff
@@ -31,3 +32,43 @@ using DetectionRobotMsg = rj_msgs::msg::DetectionRobot;
...
+		to.set_orientation(convert_from_ros(from.orientation));
+		to.set_pixel_x(convert_from_ros(from.pixel_x));
+		to.set_pixel_y(convert_from_ros(from.pixel_y));
+		to.set_height(convert_from_ros(from.height));
+		
+		return to;
+	}
+};
+
+ASSOCIATE_CPP_ROSS(SSL_DetectionRobot, DetectionRobotMsg)
```
Comment (kylestach) [inline rj_utils/include/rj_utils/conversions.hpp]:
```suggestion
ASSOCIATE_CPP_ROS(SSL_DetectionRobot, DetectionRobotMsg)
```

File: rj_utils/CMakeLists.txt [hunk 2]
```diff
@@ -38,6 +38,7 @@ ament_target_dependencies(
   rclcpp
   rj_msgs)
 
+target_dependencies(rj_utils PUBLIC rj_convert)
```
Comment (kylestach) [inline rj_utils/CMakeLists.txt]:
```suggestion
target_link_libraries(rj_utils PUBLIC rj_convert)
```

**Review by kylestach — CHANGES_REQUESTED** [review]
A few fixes, looks good!

**Review by adithya-gv — COMMENTED** [review]
Thank you! Fixed these as you said!

**Review by kylestach — APPROVED** [review]

**Review by HussainGynai — APPROVED** [review]
LGTM

**Pull request merged**